vendor/bat-native-ledger/src/bat/ledger/internal/api/api_parameters.cc

```diff
@@ -67,21 +57,22 @@ void APIParameters::OnFetch(const ledger::UrlResponse& response) {
       braveledger_response_util::ParseParameters(response, &parameters);
   if (result == ledger::Result::RETRY_SHORT) {
     RunCallbacks();
-    SetRefreshTimer(brave_base::random::Geometric(90));
+    SetRefreshTimer(base::TimeDelta::FromSeconds(90));
```

Comment:
Is there a reason to switch away from the geometric time? Was it just never necessary for these timers?

Reply:
Thanks for the double-check on this! The randomization is applied to these values in `SetRefreshTimer`.

Reply:
Got it!